assignments/prototypes.js

```diff
@@ -41,7 +41,46 @@
 
 //Test you work by uncommenting these 3 objects and the list of console logs below:
 
-/*
+function GameObject(gameobject) {
+  this.createdAt = gameobject.createdAt,
+  this.dimensions = gameobject.dimensions
+}
+
+GameObject.prototype.destroy = function() {
+  return 'Object was removed from the game.'
+}
+
+
+
+function CharacterStats(characterstats) {
+  GameObject.call(this, CharacterStats),
```

Comment:
Line 56: `CharacterStats` should match the parameter on line 55, so it should be `characterstats`! This will allow `CharacterStats` to inherit properties (`this.createdAt` and `this.dimensions`) from `GameObject`.